docs/contributor/testing-strategy.md

```diff
@@ -0,0 +1,77 @@
+# Cloud Manager Testing Strategy
+
+### Overview
+
+This testing strategy document outlines the testing activities for the Cloud Manager Kyma component. It is meant to be a guide for all specific testing-related processes. The document details the approaches, processes, and methodologies used to prioritise testing throughout the software development life-cycle.
+
+The testing strategy aims to achieve the following quality characteristics for the Cloud Manager component:
+
+* `Functional Completeness` and `Functional Correctness` by integrating validation activities into the feature acceptance process.
+* `Backward Compatibility` through regression testing and requirements management guidelines.
+* `Reliability` (with a focus on faultlessness and availability) using operational awareness logging, alerting mechanisms, and robustness system test suites.
+* `Performance Efficiency` by verifying the product against time-behavior and resource utilization test suites.
+* `Maintainability` by collecting and analyzing code quality metrics such as reusability, modularity, and software package quality.
+
+This document is refined by Test Plan documents, which define the testing approach for each particular testing activity.
+
+### Out of Scope Quality Characteristics
+
+This list of quality characteristics has not yet been defined as part of the testing strategy for the current iteration. It is yet to be refined in collaboration with respective stakeholders.
+
+| Quality Characteristic | Description                                                  | Justification                                                |
+| ---------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Compatibility          | Degree to which a product, system or component can exchange information with other products, systems or components, and/or perform its required functions while sharing the same common environment and resources. | To define a comprehensive set of compatibility requirements, we need a centralised systems engineering approach and guidelines for the interconnection and integration of Kyma components. |
+| Interaction Capability | Degree to which a product or system can be interacted with by specified users to exchange information ia the user interfaceto complete specific tasks in a variety of contexts of use. | We need to create a centralised Kyma user guide focused on user-friendliness and user engagement first. |
+| Security               | Degree to which a product or system defends against attack patterns by malicious actos and protects information and data so that persons or other products or systems have the degree of data access appropriate to their types and levels of authorisation. | We need a stakeholder to define the security expectations and to provide the testing scenarios for the different security aspects. |
```

Comment:
```suggestion
| Security               | Degree to which a product or system defends against attack patterns by malicious acts and protects information and data so that persons or other products or systems have the degree of data access appropriate to their types and levels of authorization. | We need a stakeholder to define the security expectations and to provide the testing scenarios for the different security aspects. |
```